lib/tasks/organisations.rake

```diff
@@ -1,5 +1,12 @@
+require "goodcity/organisation_populator"
```

Comment:
@trojanh There was already an organisations.rake file so it made sense to put the rake task in there.